lib/Frontend/nvqpp/ConvertExpr.cpp

```diff
@@ -659,6 +659,13 @@ bool QuakeBridgeVisitor::VisitImplicitCastExpr(clang::ImplicitCastExpr *x) {
     return pushValue(builder.create<arith::CmpIOp>(
         loc, arith::CmpIPredicate::ne, last, zero));
   }
+  case clang::CastKind::CK_FloatingToBoolean: {
+    auto last = popValue();
+    Value zero = builder.create<arith::ConstantFloatOp>(
+        loc, llvm::APFloat(0.0), cast<FloatType>(last.getType()));
+    return pushValue(builder.create<arith::CmpFOp>(
+        loc, arith::CmpFPredicate::ONE, last, zero));
```

Comment:
```suggestion
        loc, arith::CmpFPredicate::UNE, last, zero));
```

Reply:
I had stared at it for a while and had previously decided on `ONE`, but now that I've stared at it longer, and more importantly, used your other, I agree `UNE` looks better now 😄. Thanks.